learning-progression.md

```diff
@@ -132,30 +132,38 @@ basic.showIcon(IconNames.Butterfly)
 
 ### Step 5: Conditionals
 
+#### 1. Study
+
+In the English language, a statement that begins with "if" is followed by a condition. If the condition is met, the next statement follows as a logical consequence. For example, in the sentence "If it starts to rain, then I will get wet." When it starts to rain the condition is met, so the latter part of the sentence necessarily follows.
+
+A conditional statement in programming works in the same way. In the following example, the statement begins with "if," and the condition written within the parentheses is "true." Therefore, the statements inside the first block of code are run. The "else" statement can be thought of as an alternative to the "if" statement - it will only run if the condition for "if" is not met. Therefore, in this version of our code it will be ignored.
+
 ```javascript
-if (true) {
+if (true) { //This condition is always true, so the code in this block will run.
     basic.showIcon(IconNames.Heart)
     basic.pause(2000)
     basic.showIcon(IconNames.Butterfly)
-} else {
+} else { //Everything in this block is ignored because the first "if" statement was true.
     basic.showIcon(IconNames.Angry)
     basic.pause(2000)
     basic.showIcon(IconNames.Snake)
 }
-   
-if (false) {
+```
+
+Conversely, if the condition provided within the parentheses is "false," then the statements in the first block of code will be skipped. Instead, the code in the block following "else" will be run.
```

Comment:
`false` is the literal of the value of a boolean. What you mean is false w/o any quotes. In quotes, the meaning is ambiguous.

Reply:
Add a technical sentence about what the literal `false` is, mentioning "keyword", "boolean", and "value".

Reply:
At the end, say something about the "blocks" of "statements" will be executed in an either-or fashion, or "if-else" in programming languages. Say this is a "branch" in programming, meaning that the program will proceed in one of two possible directions.